sql/updates/world/2015_03_19_01_world.sql

```diff
@@ -0,0 +1,21 @@
+UPDATE `creature_template` SET `ScriptName`= 'npc_barada' WHERE `entry`= 22431;
+UPDATE `creature_template` SET `ScriptName`= 'npc_colonel_jules' WHERE `entry`= 22431;
+
+DELETE FROM `creature_text` WHERE `entry` IN(22431, 22432);
+
+-- Barada
+INSERT INTO `creature_text` VALUES (22431, 0, 0, 'It is time. The rite of exorcism will now commence...', 12, 0, 100, 0, 0, 0, 'Barada', 0);
```

Comment:
Pretty sure if you try to run these SQLs they'll give out errors since they're old and the structure has changed a lot, so we kind of have it as a standard to use full inserts, and in general, bulk queries when possible. Check some of the latest commits to see how to format sql